release_dashboard/lib/main.dart

```diff
@@ -3,41 +3,32 @@
 // found in the LICENSE file.
 
 import 'dart:io' as io;
-import 'package:conductor_core/conductor_core.dart';
-import 'package:conductor_core/proto.dart' as pb;
-import 'package:file/file.dart';
-import 'package:file/local.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
-import 'package:platform/platform.dart';
 
+import 'services/conductor.dart';
+import 'services/local_conductor.dart';
 import 'widgets/progression.dart';
 
 const String _title = 'Flutter Desktop Conductor (Not ready, do not use)';
 
-const LocalFileSystem _fs = LocalFileSystem();
-const LocalPlatform _platform = LocalPlatform();
-final String _stateFilePath = defaultStateFilePath(_platform);
-
 Future<void> main() async {
   // The app currently only supports macOS and Linux.
   if (kIsWeb || io.Platform.isWindows) {
-    throw Exception('The conductor only supports MacOS and Linux desktop');
+    throw Exception('The conductor only supports desktop on MacOS and Linux');
   }
-  final File _stateFile = _fs.file(_stateFilePath);
-  final pb.ConductorState? state = _stateFile.existsSync() ? readStateFromFile(_stateFile) : null;
 
   WidgetsFlutterBinding.ensureInitialized();
-  runApp(MyApp(state));
+  runApp(MyApp(LocalConductorService()));
```

Comment:
In the future, should we put `LocalConductorService` into a global provider, so it could be accessed across any widget?

Reply:
No. Globals are antipatterns for tests where we want to gurantee what is being used (the tool's largest technical debt is the globals, and they've spent several quarters working on removing them)